javav2/example_code/connect/Readme.md

```diff
@@ -0,0 +1,73 @@
+
+### Single action
+
+The following examples use the **ConnectClient** object:
+
+- [Create an Amazon Connect instance](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/connect/src/main/java/com/example/connect/CreateInstance.java) (createInstance command)
+- [Delete an Amazon Connect instance](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/connect/src/main/java/com/example/connect/DeleteInstance.java) (deleteInstance command)
+- [Describe the specified contact](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/connect/src/main/java/com/example/connect/DescribeContact.java) (describeContact command)
+- [Describe the specified instance](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/connect/src/main/java/com/example/connect/DescribeInstance.java) (describeInstance command)
+- [Describe the specified instance attributes](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/connect/src/main/java/com/example/connect/DescribeInstanceAttribute.java) (describeInstanceAttribute command)
+- [Describe the specified contact attributes](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/connect/src/main/java/com/example/connect/GetContactAttributes.java) (getContactAttributes command)
+- [Get historical metric data from the specified Amazon Connect instance](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/connect/src/main/java/com/example/connect/GetMetricData.java) (getMetricData command)
+- [List Amazon Connect instances](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/connect/src/main/java/com/example/connect/ListInstances.java) (listInstances command)
+- [List Amazon Connect instance phone numbers](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/connect/src/main/java/com/example/connect/ListPhoneNumbers.java) (listPhoneNumbers command)
+- [List Amazon Connect instance users.](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/connect/src/main/java/com/example/connect/ListUsers.java) (listUsers command)
+- [Search queues in an Amazon Connect instance..](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/connect/src/main/java/com/example/connect/SearchQueues.java) (searchQueues command)
+
+
+## Run the Amazon Connect Java files
+
+**IMPORTANT**
+
+The Java examples perform AWS operations for the account and AWS Region for which you've specified credentials, and you may incur AWS service charges by running them. See the [AWS Pricing page](https://aws.amazon.com/pricing/) for details about the charges you can expect for a given service and operation.
```

Comment:
The Java examples perform AWS operations for the account and AWS Region for which you've specified credentials, and you might incur AWS service charges by running them. For details about the charges you can expect for a given service and operation, see the [AWS Pricing page](https://aws.amazon.com/pricing/).